autogpt_platform/frontend/src/hooks/useCredits.ts

```diff
@@ -17,6 +17,7 @@ export default function useCredits(): {
   updateAutoTopUpConfig: (amount: number, threshold: number) => Promise<void>;
   transactionHistory: TransactionHistory;
```

Comment:
I'm removing the whole nav code since @Swiftyos has rewritten it into other files and it seems to be a dead code now.